domain/src/main/java/camp/nextstep/edu/domain/calculator/ExpressionParser.kt

```diff
@@ -0,0 +1,40 @@
+package camp.nextstep.edu.domain.calculator
+
+
+class ExpressionParser {
+
+    fun parse(
+        expression: String
+    ): List<ExpressionItem> {
+        require(expression.isNotEmpty()) { "공백은 입력할 수 없습니다." }
+
+        val items = expression.trim().split(" ")
+        require(isValidExpression(items)) { "유효하지 않은 표현식입니다." }
+
+        return toExpressionItem(items)
+    }
+
+    private fun toExpressionItem(
+        items: List<String>
+    ): List<ExpressionItem> {
+        val temp = mutableListOf<ExpressionItem>()
```

Comment:
```suggestion
        val expressionItems = mutableListOf<ExpressionItem>()
```

temp 보다는 조금 더 직관적인 이름으로 변경하는 것이 어떤가요?!